packages/distcc.rb

```diff
@@ -22,16 +22,9 @@ class Distcc < Autotools
 
   depends_on 'avahi' # R
   depends_on 'ccache' # L
-  depends_on 'gcc_dev' # L
```

Comment:
The logical deps lines are intentionally here...

Reply:
Resolved.